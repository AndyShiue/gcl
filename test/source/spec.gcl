con A, B : Int { A > 0 }
var x, y, z : Int

{ z + x * y = A * B, bnd: y }
do y /= 1 -> 
    {!
      skip
      !}
od
{ z = A * B }
