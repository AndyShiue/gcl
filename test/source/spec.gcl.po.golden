Error Struct Error ./test/source/spec.gcl:4:1
Dig Hole ./test/source/spec.gcl:4:1